aws-assume-role-policy/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "source_account_ids" {
+  type        = list(string)
+  description = "The source AWS account IDs to establish a trust relationship. Ignored if empty or not provided."
+}
+
+variable "saml_idp_arns" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>variable "saml_idp_arns" is declared but not used